src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/ExcessHet.java

```diff
@@ -16,6 +16,7 @@
 import org.broadinstitute.hellbender.utils.Utils;
 import org.broadinstitute.hellbender.utils.genotyper.ReadLikelihoods;
 import org.broadinstitute.hellbender.utils.help.HelpConstants;
+import org.broadinstitute.hellbender.utils.logging.OneShotLogger;
```

Comment:
Is there a reason why these are `OneShotLogger`s ? They're only used once.

Reply:
Yes, I wouldn't imagine it sounds like good behavior for the annotations to squak warnings at you for every annotation that passes through the engine I don't think.

Reply:
This would only fire once per annotation instance (not once per variant), and then only if you haven't provided sufficient arguments on the command line. Seems wrong to suppress those. But any rate, this import is unused now that there is a base class implementation.

Reply:
This would only fire once per annotation instance (not once per variant), and then only if you haven't provided sufficient arguments on the command line. It seems wrong to me to suppress those. But any rate, this import is unused now that there is a base class implementation, so it should be removed.

Reply:
You're right. I've removed the OneShotLoggers now.

Reply:
The (unused) imports are still here though (while you're at it there are about 5 other unused imports in there...).